graceful/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/Scalingo/go-utils/graceful
+
+go 1.14
+
+require (
+	github.com/Scalingo/go-utils/logger v0.0.0-00010101000000-000000000000
+	github.com/facebookgo/grace v0.0.0-20180706040059-75cf19382434
+	github.com/stretchr/testify v1.2.2
```

Comment:
Update to 1.6.1 as you did on some other packages here?

Reply:
Yep I'm doing it :)